Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.cpp

```diff
@@ -326,8 +326,28 @@ namespace EMotionFX
         {
             return;
         }
+        m_editor->InvalidateAll();
+        if (!m_editor->HasDisplayedNodes())
+        {
+            this->hide();
+            //m_editor->ClearInstances(true);
```

Comment:
nit: If this isn't required, please don't add commented out code (this should just be deleted if it's not necessary)